hg-fast-export.py

```diff
@@ -403,6 +449,12 @@ def verify_heads(ui,repo,cache,force,branchesmap):
 
   return True
 
+def _changectx(repo, rev):
```

Comment:
This looks like backwards compatibility code, in principle I'm against it, as with limited developer time I can't really test against anything but the latest released Mercurial. It is more honest to just bump up the minimum required Hg version when we start requiring a newer API.

I can't find any references to it, where is it used?

Reply:
Hmm, this was unused leftover code, I will remove it!